src/main/java/ch/wisv/areafiftylan/security/TeamInviteToken.java

```diff
@@ -0,0 +1,39 @@
+package ch.wisv.areafiftylan.security;
+
+import ch.wisv.areafiftylan.model.Team;
+import ch.wisv.areafiftylan.model.User;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+
+/**
+ * Created by Sille Kamoen on 9-3-16.
+ */
+@Entity
+public class TeamInviteToken extends Token {
+
+    private static final int EXPIRATION = 60 * 24 * 7;
```

Comment:
Logic for a 0 expiry is added in the transfer branch. Will update after the merge.